src/DevSoc.Eventathon/DevSoc.Eventathon.csproj

```diff
@@ -19,6 +19,7 @@
     </ItemGroup>
 
     <ItemGroup>
+        <PackageReference Include="axios.js" Version="0.19.0-beat1" />
```

Comment:
This isn't axios, you want https://www.npmjs.com/package/axios
`npm install axios --save`

Reply:
Removed it now